INSTALL.md

```diff
@@ -86,44 +86,44 @@ helm init --upgrade --service-account tiller --wait
 CF-Operator can be installed in a separate namespace:
 
 ```
-helm install --namespace cfo --name cf-operator --set "operator.watchNamespace=scf" https://s3.amazonaws.com/cf-operators/helm-charts/cf-operator-v0.4.1%2B92.g77e53fda.tgz
+helm install --namespace cfo --name cf-operator --set "operator.watchNamespace=kubecf" https://s3.amazonaws.com/cf-operators/helm-charts/cf-operator-v0.4.1%2B92.g77e53fda.tgz
 ```
 
-This allows us to restart the operator, because it is not affected by webhooks. We can also delete the SCF deployment namespace to start from scratch, without redeploying the operator.
+This allows us to restart the operator, because it is not affected by webhooks. We can also delete the KUBECF deployment namespace to start from scratch, without redeploying the operator.
```

Comment:
Okay that's just inconsistent now.

Reply:
Fixed with commit 052140422ca2bcf39734b60d99587871dd7f920a.